test/PlaygroundTransform/Inputs/not.py

```diff
@@ -0,0 +1,13 @@
+import shlex
+import subprocess
+import sys
+
+if len(sys.argv) < 2:
+    print("Too few args to " + sys.argv[0])
+    sys.exit(0)
+
+try:
+    subprocess.check_call(shlex.split(' '.join(sys.argv[1:])))
```

Comment:
Wait, no, this is incorrect. Either you pass through the arguments as given (`sys.argv[1:]`) and don't quote when calling not.py, or you use shlex.split on a single argument (`shlex.split(sys.argv[1])`) and *do* quote. Don't do both.

Reply:
I wanted to allow for

not.py ls -l

or

not.py "ls -l"

In case it gets reused.

Whats wrong with that?

Reply:
It breaks `not.py ls "file with spaces.txt"`. It's better if the client knows up front what they have to escape.

Reply:
doe! this landed. I will do another PR asap to address this @jrose-apple

Reply:
Addressing this in PR #23637